pwa/src/components/handlers/handlerForm.tsx

```diff
@@ -16,15 +16,22 @@ import {
 } from "@conductionnl/nl-design-system/lib";
 import {isLoggedIn} from "../../services/auth";
 import FlashMessage from 'react-flash-message';
-import {MultiDimensionalArrayInput} from "../common/multiDimensionalArrayInput";
-import ElementCreationNew from "../common/elementCreationNew"
+import { ArrayInputComponent } from "../common/arrayInput";
```

Comment:
Ik verwacht dat dit in een git conflict is aangepast, je hebt echter niet de juiste optie gekozen.

De nieuwe (en gewenste) versie is de `ElementCreationNew` die je nu niet meer include. A.u.b. hele bestand checken en zorgen dat het nieuwe element weer included wordt.

Note: goed testen!